provisioner/puppet-masterless/provisioner.go

```diff
@@ -92,7 +92,7 @@ var guestOSTypeConfigs = map[string]guestOSTypeConfig{
 		stagingDir: "C:/Windows/Temp/packer-puppet-masterless",
 		executeCommand: "cd {{.WorkingDir}} && " +
 			"{{.FacterVars}} && " +
-			"puppet apply --verbose --modulepath='{{.ModulePath}}' " +
+			"{{if ne .PuppetBinDir \"\"}}{{.PuppetBinDir}}/{{end}}puppet apply --verbose --modulepath='{{.ModulePath}}' " +
```

Comment:
looks good. Should this also get added to the Unix template?

Reply:
Also might be worth splitting this into multiple lines.

You can also use backticks (`) instead of double quotes (`"`) to create a raw string, which will prevent you from needing to escape the string. see https://play.golang.org/p/qlpO7CdTD_

Reply:
Oops, missed that. Yes it should be in the Unix template.

Thanks for the tip re: backticks, looks nice but current style is consistent with the puppet-server module. I don't mind either way.

Re: Splitting across lines, I think it is worth discussing whether or not `executeCommand` should actually become a slice of `executeCommands` that are executed in series, it seems like it would be more scalable in the long run but would require a bit of work.